package.json

```diff
@@ -17,15 +17,15 @@
   },
   "homepage": "https://github.com/magalhas/express-react-engine",
   "peerDependencies": {
-    "react": "^0.14.0",
-    "react-dom": "^0.14.0"
+    "react": "^15.0",
```

Comment:
perhaps this could be `"react": "14.x || 15.x"` and so on, so that it can remain compatible with both versions?